src/containers/DeveloperPortalProjects/components/Breadcrumb/Breadcrumb.scss

```diff
@@ -0,0 +1,94 @@
+@import 'src/styles/mixins';
+@import 'styles/font-mixins';
+@import '../../shared';
+
+.Breadcrumb {
+  align-items: center;
+  background-color: white;
+  display: flex;
+  flex-wrap: wrap;
+  min-height: $breadcrumbs-height;
+
+  @media (max-width: 1366px) {
+    padding: 0 48px;
+  }
+
+  @media (max-width: 992px) {
+    padding: 12px 24px;
+  }
+
+  &__link-container {
+    align-items: center;
+    color: var(--color-gray-400);
+    display: flex;
+
+    @media (max-width: 768px) {
+      margin: 12px 0;
+    }
+  }
+
+  &__link {
+    @extend %h4;
+    color: var(--color-gray-400);
+    font-weight: var(--font-weight-regular);
+    line-height: 14px; // allow text to be centered by setting line-height to be the same as font size
```

Comment:
can just put `line-height: 1` haha

Reply:
I did not make any changes to Breadcrumb and SideMenu component. These can be done later, separately.